REUSE.toml

```diff
@@ -0,0 +1,46 @@
+# ~~~
+# Copyright (c) 2024 The Khronos Group Inc.
+# Copyright (c) 2024 RasterGrid Kft.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ~~~
+version = 1
+SPDX-PackageName = "Vulkan SC Emulation"
+SPDX-PackageSupplier = "The Khronos Group, Inc."
+SPDX-PackageDownloadLocation = "https://github.com/KhronosGroup/VulkanSC-Emulation"
+
+[[annotations]]
+path = [
+    "tests/pcc/data/**",
+    ".clang-format",
+    "icd/generated/.clang-format",
+    "icd/icd_shadow_stack.cpp",
+    "icd/vksconvk.json.in",
+    "scripts/known_good.json",
+    "tests/icd/vkmock_icd.json.in",
+    "tests/pcc/caselist.pcc.txt"
+]
+precedence = "aggregate"
+SPDX-FileCopyrightText = "2024 The Khronos Group, Inc."
+SPDX-License-Identifier = "Apache-2.0"
+
+[[annotations]]
+path = [
+    "**.md",
+    ".gitattributes",
+    ".gitignore"
+]
+precedence = "aggregate"
+SPDX-FileCopyrightText = "2024 The Khronos Group, Inc."
+SPDX-License-Identifier = "CC-BY-4.0"
+
+[[annotations]]
+path = "scripts/update_deps.py"
+precedence = "aggregate"
+SPDX-FileCopyrightText = [
+    "2017 The Glslang Authors. All rights reserved.",
+    "2018-2023 Valve Corporation",
+    "2018-2023 LunarG, Inc.",
+    "2023-2023 RasterGrid Kft."
+]
+SPDX-License-Identifier = "Apache-2.0"
```

Comment:
I believe this whole block could be removed as well, because update_deps.py has also been updated to include SPDX style license information.

Reply:
Now it has.